app/helpers/schema_helper.py

```diff
@@ -1,42 +1,43 @@
 from app.validation.error_messages import error_messages
 
 
-class SchemaHelper(object):  # pylint: disable=too-many-public-methods
+class QuestionnaireSchema(object):  # pylint: disable=too-many-public-methods
```

Comment:
- We should rename this `questionnaire_schema.py` and move it to `app/questionnaire`
- `survey_json` should be `questionnaire_json` in the `__init__` below